contracts/l2/L2DAIWormholeBridge.sol

```diff
@@ -113,6 +131,7 @@ contract L2DAIWormholeBridge is OVM_CrossDomainEnabled {
 
   function flush(bytes32 targetDomain) external {
     uint256 daiToFlush = batchedDaiToFlush[targetDomain];
+    require(daiToFlush > 0, "L2DAIWormholeBridge/zero-dai-flush");
```

Comment:
WDYT about leaving a comment here that we don't need to check against validDomains b/c batchedDaiToFlush will be always 0 for them

Reply:
This is not strictly true if we open a target domain then later close it. I think we can omit the comment in the same way we don't mention that we can still flush when the bridge is closed.